### PR TITLE
release: sourcegraph@5.3.1

### PR DESCRIPTION
**Pull request by BolajiOlajide**
This pull request is part of the Sourcegraph 5.3.1 release.

* [Release batch change](https://sourcegraph.sourcegraph.com/organizations/sourcegraph/batch-changes/release-sourcegraph-5.3.1)
* [Tracking issue](https://github.com/sourcegraph/sourcegraph/issues/60691)

### Test plan

CI checks in this repository should pass, and a manual review should confirm if the generated changes are correct.

### CODE REVIEW

**Review by evict — APPROVED** [review]

**Comment by BolajiOlajide** [comment]
@MaedahBatool Do you want to take a look at this?

File: docs/CHANGELOG.mdx [hunk 1]
```diff
@@ -2936,7 +2936,7 @@ This is `3.12.8` release with internal infrastructure fixes to publish the docke
 ### Fixed
 
 - The `/.auth/saml/metadata` endpoint has been fixed. Previously it panicked if no encryption key was set.
-- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
+- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:5.3.1` will need to manually modify their `docker run` command to use `sourcegraph/server:5.3.1` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```
Comment (BolajiOlajide) [inline docs/CHANGELOG.mdx:2939]:
```suggestion
- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```

Comment (BolajiOlajide) [reply docs/CHANGELOG.mdx:2939]: I thought I removed this file from the release tool. I have to check why the tool still tries to update this file.

File: docs/versioned/5.2/CHANGELOG.mdx [hunk 2]
```diff
@@ -2896,7 +2896,7 @@ This is `3.12.8` release with internal infrastructure fixes to publish the docke
 ### Fixed
 
 - The `/.auth/saml/metadata` endpoint has been fixed. Previously it panicked if no encryption key was set.
-- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
+- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:5.3.1` will need to manually modify their `docker run` command to use `sourcegraph/server:5.3.1` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```
Comment (BolajiOlajide) [inline docs/versioned/5.2/CHANGELOG.mdx:2899]:
```suggestion
- The version updating logic has been fixed for `sourcegraph/server`. Users running `sourcegraph/server:3.11.1` will need to manually modify their `docker run` command to use `sourcegraph/server:3.11.4` or higher. [#7442](https://github.com/sourcegraph/sourcegraph/issues/7442)
```

Gaaaah!

**Pull request merged**